Jenkinsfile

```diff
@@ -133,7 +133,24 @@ waitUntil {
 		build job: downstreamJob, parameters: [[$class: 'StringParameterValue', name: 'WORKFLOW_CLI_SHA', value: git_commit]]
 		true
 	} catch(error) {
-		 input "Retry the e2e tests?"
-		 false
+			node('linux') {
+				withCredentials([[$class: 'StringBinding', credentialsId: '8a727911-596f-4057-97c2-b9e23de5268d', variable: 'SLACKEMAIL']]) {
+					mail body: """<!DOCTYPE html>
+<html>
+<head>
+<meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+</head>
+<body>
+<div>Author: ${env.CHANGE_AUTHOR}<br/>
+Branch: ${env.BRANCH_NAME}<br/>
+Commit: ${env.CHANGE_TITLE}<br/>
+<a href="${env.BUILD_URL}input/">Click here</a> to restart e2e.</p>
```

Comment:
I'm thinking that we might want to add a line `<a href="${env.BUILD_URL}console/">Click here</a> to view build logs.</p>` because sometimes it is hard to tell why e2e failed to make the decision about whether or not to restart it.